polaris

```diff
@@ -28,6 +28,7 @@ if [ ! -d "${dir}"/polaris-venv ]; then
   python3 -m venv "${dir}"/polaris-venv
   . "${dir}"/polaris-venv/bin/activate
   pip install -r regtests/requirements.txt
```

Comment:
I ack this is how we install dependencies here but I was hesitant to add these requirements to the `regtests` section since idk if they are actually necessary there.
Perhaps creating a regular `requirements.txt` is in order?